src/family/family.service.spec.ts

```diff
@@ -57,4 +60,33 @@ describe('FamilyService', () => {
       expect(family).toEqual(familyStub);
     });
   });
+
+  describe('create', () => {
+    it('should be defined', () => {
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      expect(service.create).toBeDefined();
+    });
+
+    it('should called with correct argument', async () => {
+      const data: CreateFamilyInput = { name: faker.name.fullName() };
+      const method = jest
+        .spyOn(prisma.family, 'create')
+        .mockResolvedValue(familyStub);
+      await service.create(data);
+
+      expect(method).toHaveBeenCalledWith({
+        data: { ...data, slug: slug(data.name) },
+      });
+    });
+
+    it('should throw unrecognized error', async () => {
+      jest
+        .spyOn(prisma.family, 'create')
+        .mockRejectedValue(new Error('Wrong Input'));
+
+      await expect(
+        service.create({ name: faker.name.fullName() }),
+      ).rejects.toThrow();
+    });
+  });
```

Comment:
I think it's better to write a test case for the expected result too

Reply:
Done